hack/quickstart/kubelet.master

```diff
@@ -11,7 +11,8 @@ ExecStartPre=/bin/mkdir -p /etc/kubernetes/cni/net.d
 ExecStartPre=/bin/mkdir -p /etc/kubernetes/checkpoint-secrets
 ExecStartPre=/bin/mkdir -p /etc/kubernetes/inactive-manifests
 ExecStartPre=/bin/mkdir -p /var/lib/cni
-ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/run/kubelet-pod.uuid
+ExecStartPre=/bin/mkdir -p /var/cache/rkt
```

Comment:
nit: need to change this line too.

alternatively can probably just use /var/cache/kubelet-pod.uuid w/out subdirectory.